src/chart/MultiChart.js

```diff
@@ -143,7 +143,10 @@
         if (arguments.length) {
             var context = this;
             _.click = function (row, column, selected) {
-                context.click(row, column, selected);
+                context.click.apply(context, arguments);
```

Comment:
```context.click.apply(context, arguments);``` FYI this means call the "click" event on a base class passing in the current argument list - so row, column + selected all get passed to the base click event.